### PR TITLE
fix(ui): Double clicking controls no longer enters fullscreen

### PR DESCRIPTION
**Pull request by Thomasvdam**
Also works for double taps.

Closes meisterplayer/meisterplayer#43

### CODE REVIEW

**Review by FranklinWaller — APPROVED** [review]

File: src/js/bottom/BottomBar.js [hunk 1]
```diff
@@ -81,10 +81,16 @@ class BottomBar extends BaseElement {
...
         // Clicks inside the control bars shouldn't play/pause the player.
-        this.element.addEventListener('click', (e) => {
-            const clickEvent = e || event;
-            clickEvent.stopPropagation();
-        });
+        this.element.addEventListener('click', this.stopPointerPropagation);
+
+        // Double clicks/taps inside the controls shouldn't trigger fullscreen.
+        this.element.addEventListener('dblclick', this.stopPointerPropagation);
+        this.element.addEventListener('touchstart', this.stopPointerPropagation);
```
Comment (ivo-toby) [inline src/js/bottom/BottomBar.js]: Shouldn't these bind to this ? Or enclose in an arrowmethod?

Comment (FranklinWaller) [reply src/js/bottom/BottomBar.js]: It doesn't use 'this' in stopPointerPropagation but i agree to keep consistency (Plus it's part of the same class) it should bind to this

**Review by ivo-toby — CHANGES_REQUESTED** [review]
Bind callbacks to this or wrap callbacks into arrow function

**Comment by Thomasvdam** [comment]
Bound the callbacks, good catch!

**Review by ivo-toby — APPROVED** [review]

**Pull request merged**